Source/src/ui/WindowHierarchy.cpp

```diff
@@ -28,27 +28,43 @@ void Hachiko::WindowHierarchy::CleanUp()
 
 void Hachiko::WindowHierarchy::DrawHierarchyTree(const GameObject* game_object)
 {
-    DrawChildren(game_object);
+    for (const auto go : game_object->children)
+    {
+       RecursiveDraw(go);
+    }
+
     if (!App->input->GetMouseButton(SDL_BUTTON_LEFT))
     {
         dragged_object = nullptr;
     }
 }
 
-void Hachiko::WindowHierarchy::DrawChildren(const GameObject* game_object)
+bool Hachiko::WindowHierarchy::RecursiveDraw(GameObject* game_object)
 {
-    for (const auto go : game_object->children)
+    bool stop = false;
+
+    const char* str = strstr(game_object->name.c_str(), "$AssimpFbx$");
```

Comment:
can we move this magic string to a constant?

Reply:
Yeah of coourse